src/Tests/Microsoft.Diagnostics.Monitoring.Tool.UnitTests/MockSystemClock.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.AspNetCore.Authentication;
+using System;
+
+namespace Microsoft.Diagnostics.Monitoring.Tool.UnitTests
+{
+    internal class MockSystemClock : ISystemClock
+    {
+        private DateTimeOffset _utcNow = DateTime.UtcNow;
```

Comment:
What's the reasoning behind using `DateTime.UtcNow` here and `DateTimeOffset.UtcNow` for `RealSystemClock`? I see they're both being assigned to `DateTimeOffset` types, but I don't have a good understanding of why we want `DateTime` for the mocked clock and `DateTimeOffset` for the real clock.

Reply:
I think they are equivalent when you dig down into them. I'll change the mock to use DateTimeOffset to be consistent.